### PR TITLE
Fixes™

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

File: rpcs3/rpcs3qt/game_list_frame.cpp [hunk 1]
```diff
@@ -1114,62 +1114,54 @@ int game_list_frame::IsValidFile(const QMimeData& md, QStringList* dropPaths)
...
-		if (suffix == "pkg")
-		{
 			dropType = DROP_PKG;
 		}
-		else if (suffix == "pup")
-		{
-			dropType = DROP_PUP;
-		}
-		else if (suffix == "rap")
+		else if (info.suffix() == "rap")
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/game_list_frame.cpp]: why not use .toLower() here ?

Comment (Nekotekina) [inline rpcs3/rpcs3qt/game_list_frame.cpp]: It shall be lowercase.

**Comment by wildanaisy** [comment]
dddd

**Comment by wildanaisy** [comment]
how to use github

**Comment by SaltyCucumber** [comment]
@wildanaisy best practice is to use brain.exe first, if that returns error 404 then seek #help in discord

**Pull request merged**